metabrainz/templates/users/profile.html

```diff
@@ -75,59 +75,84 @@ <h3>{{ _('Contact information') }}</h3>
   {% if current_user.is_commercial %}
     {% if current_user.state in ('active', 'limited') and current_user.good_standing %}
       <h3>{{ _('Data use permission granted') }}</h3>
-      <p>
-        {{ _('You have permission to use any of the data published by the MetaBrainz
-        Foundation. This includes data dumps released under a Creative Commons
-        non-commercial license. Thank you for your support!') }}
-      </p>
-      <p>
-        {{ _('Note 1: If your support falls behind by more than 60 days, this
-        permission may be withdrawn. You can always check your current permission
-        status on this page.') }}
-      </p>
-      <p>
-        {{ _('Note 2: The IP addresses from which replication packets for the Live Data Feed are downloaded are logged.') }}
-      </p>
+        <div style="font-size: 13pt">
+        <p>
+          <b>Your support agreement has been completed -- thank you!</b>
+        </p>
+        <p>
+          {{ _('You have permission to use any of the data published by the MetaBrainz
+          Foundation. This includes data dumps released under a Creative Commons
+          non-commercial license. Thank you for your support!') }}
+        </p>
+        <p>
+          {{ _('Note 1: If your support falls behind by more than 60 days, this
+          permission may be withdrawn. You can always check your current permission
+          status on this page.') }}
+        </p>
+        <p>
+          {{ _('Note 2: The IP addresses from which replication packets for the Live Data Feed are downloaded are logged.') }}
+        </p>
+      </div>
     {% else %}
-      <h3>{{ _('Limited data use permission granted') }}</h3>
-      <p>
-        {{ _('You do not currently have permission to use data released under a Creative Commons non-commercial license for commercial purposes.') }}
-      </p>
-      <p>
-        {{ _('Reason:') }}
+      <h3>Limited/no data use permission granted</h3>
```

Comment:
it the i18n stuff (`_('…')`) removed on purpose, on this line and a few others below ?

Reply:
Well, we will certainly need i18n support for the parts of the site that will be used for standard users to log in and whatnot, especially once we move to MeB accounts. That said, the business-facing bits being all English make sense to me.

Reply:
Its the business and more importantly legal aspects that are super $$$ to fully translate. I'm totally ok, for having the login pages localized.